lib/tape/ansible_runner.rb

```diff
@@ -7,13 +7,13 @@ class AnsibleRunner < ExecutionModule
     proc {ansible '-t configure_dj_runner -e force_dj_runner_restart=true'},
     "Configures and restarts the delayed job runner"
   action :restart_unicorn,
-    proc {ansible '-t unicorn_restart'}, 
+    proc {ansible '-t unicorn_restart'},
     "Restarts the unicorns running on the app servers"
   action :stop_unicorn,
-    proc {ansible '-t unicorn_stop -e kill_unicorn=true'}, 
+    proc {ansible '-t unicorn_stop -e kill_unicorn=true'},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.